### DISCUSSION TITLE
重复key的bug

### DISCUSSION THREAD

**Discussion by i7ian** [Mrs4s/go-cqhttp#2216] [Ideas]
接收匿名消息时会存在两个sub_type，应该以后一个为准，很多json解析过程中会保留第一个或者后一个，会导致判断错误

{"post_type":"message","message_type":"group","time":1686577487,"self_id":xxxx,**"sub_type":"normal",**"anonymous":{"flag":"oVoKgISAcvBZ4svmcT73Eu11/Cl620FCm+Z//DPtGPQ5gEgkXFG8FA==|净坛使者","id":80000000,"name":"净坛使者"},"message_seq":73657,**"sub_type":"anonymous"**,"message_id":-1882889743,"sender":{"age":0,"area":"","level":"","nickname":"匿名消息","sex":"unknown","user_id":80000000},"user_id":80000000,"font":0,"group_id":xxxx,"message":"[CQ:face,id=314,type=sticker]","raw_message":"[CQ:face,id=314,type=sticker]"}

**Comment by xiangxiangxiong9** [comment]
已复现 [考虑转换为issue]

**Comment by i7ian** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
